src/WorkItemMigrator/WorkItemImport/Agent.cs

```diff
@@ -57,7 +57,7 @@ public WorkItem CreateWorkItem(string type, bool suppressNotifications, DateTime
             return _witClientUtils.CreateWorkItem(type, suppressNotifications, createdDate, createdBy);
         }
 
-        public bool ImportRevision(WiRevision rev, WorkItem wi, Settings settings)
+        public bool ImportRevision(WiRevision rev, WorkItem wi, Settings settings, bool isFinalRevision)
```

Comment:
Where is the parameter `isFinalRevision` actually used? It does not seem to be in use.

Reply:
Well spotted.  I've got a second feature that I'm currently re-implementing in the community version.  It _may_ benefit from having this here, but I'm not sure yet.  So if you can hang on an extra day, I'll let you know.

Reply:
Ok! :)

Reply:
I've removed this unused parameter.